pola/templates/base-landing.html

```diff
@@ -28,50 +28,7 @@
 
 <body>
 
-    <div class="navbar navbar-default">
-        <div class="container-fluid">
-            <!-- Brand and toggle get grouped for better mobile display -->
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-navbar-collapse-1" aria-expanded="false">
-                    <span class="sr-only">{% trans "Toggle navigation" %}</span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                <a class="navbar-brand" href="/">{% trans "Pola" %}</a>
-            </div>
-
-            <!-- Collect the nav links, forms, and other content for toggling -->
-            <div class="collapse navbar-collapse" id="bs-navbar-collapse-1">
-                <ul class="nav navbar-nav">
-                    <li><a href="{% url 'home' %}">{% trans "Home" %}</a></li>
-                    <li><a href="{% url 'about' %}">{% trans "About" %}</a></li>
-                </ul>
-
-                <ul class="nav navbar-nav navbar-right">
-                    {% if request.user.is_authenticated %}
-                    <li><a href="{% url 'home-cms'  %}">{% trans "CMS" %}</a></li>
-                    {% else %}
-                    <li><a href="{% url 'account_login' %}">{% trans "Log In" %}</a></li>
-                    {% endif %}
-                </ul>
-            </div>
-        </div>
-    </div>
-
-    <div class="container">
-
-        {% if messages %}
```

Comment:
Tutaj psujesz dziedziczenie templatki. Ten tekst, który jest ustawiany nigdy nie jest wyświetalny, ale inne templatki z niego dziedziczą. Zrobiłem inny commit, ayb pokazać, jak to powinno wygladać. Jeśli jednak nie chcesz dziedziczenia to powinieneś poprawić wyświetlaną templatkę w widoku.